CHANGELOG.md

```diff
@@ -1,43 +1,94 @@
+- Toggle between listing streams & topics in a stream easily using <kbd>t</kbd>
+- Examine who reacted to a message using the information menu accessible using <kbd>i</kbd>
+- Mute a selected stream from within zulip-term using <kbd>m</kbd>
+- Various areas now marked with shortcut keys in the UI, not just in the help menu (<kbd>?</kbd>)
 
 ### Interactivity improvements
-- Add feature to edit sent messages using <kbd>e</kbd> hotkey.
+- The 'no_autohide' option is now the default, with both left and right panels always shown
+- Added <kbd>e</kbd> to edit your own sent message content & subject/topic
+- Update message content and subject/topic appropriately from server (#401)
+- Change views as per webapp if the edited message subject/topic changes
+- Only mark messages read, if actively move across to message list and select them
+- Added <kbd>z</kbd> hot/shortcut key to toggle between stream/topic or all-private/specific-person narrows
+- Added <kbd>t</kbd> to toggle between streams & list of all topics in selected stream
+- Added <kbd>m</kbd> to mute/unmute selected stream, with confirmation popup
+- Added <kbd>i</kbd> to show further information of selected message (including reactions)
 - Use <kbd>?</kbd> to show *and* hide help menu (<kbd>esc</kbd> continues to exit)
-- Add additional shortcut key for sending messages (<kbd>ctrl</kbd>+<kbd>d</kbd>)
+- Add additional shortcut key for sending messages (<kbd>ctrl</kbd>+<kbd>d</kbd>) (#176)
+- Add hint in footer of how to copy text, if attempt to select text (with mouse) in terminal
+- Stream search is now case-insensitive
+- Add command-line option to check version (#280)
 - Allow <kbd>f</kbd> and <kbd>P</kbd> shortcut keys to work from side panels (narrow starred & private messages)
 - Warn on startup, if specified theme is lacking current required styles
 - Upon unexpected crash, exit cleanly and log traceback to `zulip-terminal-tracebacks.log`
-- Add <kbd>z</kbd> hot/shortcut key to toggle between stream/topic or all-private/specific-person narrows
 
 ### Visual improvements
 - Right-align unread-counts in left & right panels (as in webapp)
 - Remove intrusive flashing cursor in side panels
-- Truncate text in left & right panels cleanly with '..', avoiding text overflow
-- Show user status as `active` (Green), `idle` (Yellow) or `offline` (White) using different colors.
+- Truncate text in left & right panels cleanly with '…', avoiding text overflow (#237)
+- Show users' status as `active` (Green), `idle` (Yellow) or `offline` (White) using different colors.
+- Update user list every minute, not just at startup (#330)
+- Sort user list alphabetically, case-independently (#264)
 - Further improvement/reordering of shortcut keys in README & help menu (<kbd>?</kbd>)
-- Improve styling of help menu, and how the menu scales with application width & height
+- Improve styling of help menu, and how it menu scales with application width & height (#297, #307)
```

Comment:
Would `it` be grammatically right here?

Reply:
Thanks for catching this - I meant to drop the second use of 'menu'.